share/locale/it.json

```diff
@@ -16,10 +16,12 @@
     "gui_choose_items": "Scegli",
     "gui_share_start_server": "Inizia la condivisione",
     "gui_share_stop_server": "Ferma la condivisione",
-    "gui_copy_url": "Copia lo URL",
+    "gui_copy_url": "Copia l' URL",
     "gui_downloads": "Downloads:",
     "gui_canceled": "Cancellati",
     "gui_copied_url": "URL Copiato nella clipboard",
     "gui_please_wait": "Attendere prego...",
     "zip_progress_bar_format": "Elaborazione files: %p%"
+    "gui_share_quit_warning": "Stai per inviare dai file. Sei sicuro di voler uscire da OnionShare?"
```

Comment:
Hello @DogeIlMeme, thanks for your translations!

Can you please add a comma `,` at the end of the line above (`Elaborazione files`), as well as this line, but *not* your final line. This is needed in JSON syntax - every line must end with a comma, except for the last line.

Thanks!